src/pages/en/legal/cozy-privacy-policy.mdx

```diff
@@ -5,74 +5,53 @@ description: The privacy policy for the Cozy Discord bots.
 
 The Quilt Project ("Us", "We", "Our") is the data controller for the data collected through the Cozy Discord bots.
 
-# Data Collected
+## Data Collected
 
-## Applications
-
-When you submit an application to join the server, we save your User ID along with some metadata. Your answers aren't
-stored.
-
-We use this data to maintain a list of previous applications that have been submitted by each user, which allows us to
-look them up on Discord when we need to review them.
-
-## Filter Events
+### Filter Events
 
 When you trigger an automatic filter, we record your User ID and the location of the message that triggered it.
 The message content isn't saved.
 
-We use this data to conduct analytics.
+We use this data to allow moderators to review and action messages that are triggered by the automatic filters.
 
-## Threads
+### Threads
 
 If you are designated as the owner of a thread by creating it, or having the current owner transfer it to you,
 your User ID is saved alongside the thread ID.
 
 We use this data to allow you to manage the threads you own.
 
-## Suggestions
-
-### Voting
-
-When you vote positively or negatively, we save your User ID alongside the suggestion and your vote.
-
-We use this data to make sure you can't vote more than once.
-
-### Creating a suggestion
-
-When you create a suggestion, we save your User ID, Username, Avatar Url, Message and whether it was proxied or not.
-
-We use this data to display and keep track of the suggestions. This data can be seen on the public analytics dashboard, without the User ID or Username.
-
-## User Flags
+### User Flags
 
 We save three flags about you:
-- Has Used PluralKit: whether you have used PluralKit previously or not. We use that to log new usages of PluralKit for
-review by the moderators.
+- Has Used PluralKit: whether you have used PluralKit previously or not. We use that to log new usages of PluralKit for review by the moderators.
 - Auto Publish: whether your messages sent in a news channel should be automatically published.
 - Sync Nicknames: whether your nickname should be synced between Quilt guilds.
 
-# Data Retention
+## Data Retention
 
 Data is retained indefinitely. We do not delete any data unless you request it.
 
-# Data Governance
+## Data Governance
+
+Database access is limited to the minimum of The Quilt Project benevolent workers required to operate the service. We do not share any data with third parties. We do not sell any data.
 
-Database access is limited to the minimum of The Quilt Project benevolent workers required to operate the service. We do not share any data with third parties.
-We do not sell any data. Your data isn't shared with any contracted sub-processors.
+Data is stored in the United States of America, with the following processors:
+- [Backblaze](https://www.backblaze.com)
+- [Capitol Hosting Solutions](https://chs.gg)
```

Comment:
What steps are taken to ensure the level of protection is equivalent to the ones provided in a European Union country?

Reply:
Both companies are in the USA and thus fall under the EU-US Umbrella Agreement. Backblaze is already part of the [Data Privacy Framework](https://www.dataprivacyframework.gov/list), which ensures it has the required level of protection. CHS has only submitted its application and is therefore a bit more of a gray area until said application is processed, but they already fulfill their obligations in terms of data protection.